egs/ami/s5b/local/chain/multi_condition/run_tdnn.sh

```diff
@@ -0,0 +1,280 @@
+#!/bin/bash
+
+# This is a chain-training script with TDNN neural networks.
+# This script inherits from local/chain/run_tdnn.sh, but adding
+# the reverberated IHM data into the train set.
+# This script obtains better results on both IHM and SDM tasks.
+
+# Please see RESULTS_* for examples of command lines invoking this script.
+
+# local/chain/multi_condition/run_tdnn.sh --mic ihm --train-set train_cleaned --gmm tri3_cleaned &
+# local/chain/multi_condition/run_tdnn.sh --mic sdm1 --use-ihm-ali true --train-set train_cleaned --gmm tri3_cleaned &
+
+
+set -e -o pipefail
+
+# First the options that are passed through to run_ivector_common.sh
+# (some of which are also used in this script directly).
+stage=1
+mic=ihm
+nj=30
+min_seg_len=1.55
+use_ihm_ali=true
```

Comment:
As you are already in the multi-condition directory you can make add_reverberated_Data compulsary. Also if you are using add_reverberated_data option use_ihm_ali would be true*. So you could eliminate both these options.

*As you are already assuming availability of IHM data it does not make sense to not use it for alignments.